features/schema.py

```diff
@@ -210,27 +211,41 @@ def resolve_parents(self: models.Feature, info, **kwargs):
     def resolve_children(self: models.Feature, info, **kwargs):
         return self.children.all()
 
+    @classmethod
+    def get_queryset(cls, queryset, info):
+        return queryset.select_related("source_type", "category",).prefetch_related(
+            "category__translations",
+            "contact_info",
+            "children",
+            "images",
+            "images__license",
+            "images__license__translations",
+            "opening_hours_periods",
+            "opening_hours_periods__opening_hours",
+            "opening_hours_periods__translations",
+            "parents",
+            "tags",
+            "tags__translations",
+            "translations",
+        )
```

Comment:
Should this also prefetch the overrides?

Reply:
It should, but currently the way we resolve e.g. the `name` field in the `resolve_name` method doesn't allow us to take into account any prefetching that we do here since in the method we're filtering on a queryset which makes a new query to the DB. :sob:

That said queries done in `resolve_name` and `resolve_modified_at` could maybe be optimized so that we could reduce the amount of additional queries when resolving features.

Reply:
Ah, okay. Well, let's worry about optimization separately. IFF we have to 😉